sdk/schemaregistry/schema-registry/package.json

```diff
@@ -30,7 +28,7 @@
     "test:node": "npm run build:test && npm run integration-test:node",
     "test": "npm run build:test && npm run unit-test && npm run integration-test",
     "unit-test:browser": "karma start --single-run",
-    "unit-test:node": "mocha --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 1200000 --full-trace dist-test/index.node.js",
+    "unit-test:node": "mocha -r esm -r ts-node/register --timeout 50000 --reporter ../../../common/tools/mocha-multi-reporter.js --colors --exclude \"test/**/*.browser.ts\" \"test/**/*.ts\"",
```

Comment:
Why is the timeout value changed?

Reply:
The operations in this service are fast and do not justify the higher timeout value